<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QDialog" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="profileWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>721</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#profileWidget{
	background-color: rgb(86, 86, 86);
}</string>
   </property>
   <widget class="QLabel" name="profileLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#profileLabel{

color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="profileNameLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#profileNameLabel{

color: rgb(255, 170, 0);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="phonelabelTitle">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>461</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#phonelabelTitle{
 color:white;

}</string>
    </property>
    <property name="text">
     <string>Phone:</string>
    </property>
   </widget>
   <widget class="QLabel" name="phonelabelValue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>461</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#phonelabelValue{

	color: rgb(255, 170, 0);

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#uploadButton{
border-image: url(:/images/C:/Users/asus/Downloads/images.png)
	
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="userProfileView">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#userProfileView{
background-color:rgb(218, 218, 218)

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="namelabelTitle">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>420</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#namelabelTitle{
 color:white;
}</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
